src/cargo/ops/cargo_package.rs

```diff
@@ -28,6 +28,7 @@ use tar::{Archive, Builder, EntryType, Header, HeaderMode};
 use tracing::debug;
 use unicase::Ascii as UncasedAscii;
 
```

Comment:
As for commit titles, strictly speaking this is not a refactor as it has user-visible changes